portal/templates/redesign/teach_new.html

```diff
@@ -111,19 +111,11 @@ <h1 class="banner__text--primary">Educate</h1>
     $(document).ready(function(){
         invokeColorbox();
     });
+</script>
 
-    $(window).scroll(function() {
-        var currentScroll = $(window).scrollTop();
-        if (currentScroll >= 343) {
-            $('.sub-nav--hidden').css({
-                'display': 'block'
-            });
-        } else {
-            $('.sub-nav--hidden').css({
-                'display': 'none'
-            });
-        }
-    });
+<script type="text/javascript" src="{% static 'portal/js/sticky_subnav.js' %}"></script>
+<script>
+    var scrollToTop = 343;
```

Comment:
So, about the setting of this var: this looks like a magic number.  It works now, but in fact it's dependent on the conditions on the page at this time.  If you change that, this number will no longer work.  As with Sass vars, it's probably a good idea to make the setting of this var explicitly pull the number it needs from wherever it needs it. 

eg.  If 343 px is the position of some page element, use JS to query for the position of the page element and use _that_, rather than assuming the position will always be what it is now.